packages/x-charts/src/ChartsGrid/ChartsHorizontalGrid.tsx

```diff
@@ -25,7 +25,7 @@ export function ChartsGridHorizontal(props: ChartsGridHorizontalProps) {
     <React.Fragment>
       {yTicks.map(({ value, offset }) => (
         <GridLine
-          key={`horizontal-${value.getTime?.() ?? value}`}
+          key={`horizontal-${value?.getTime?.() ?? value}`}
```

Comment:
Doesn't this mean our types are wrong? If value can be `null` or `undefined`, shouldn't we have gotten a TypeScript error?